client/signup/steps/p2-email-confirmed/index.jsx

```diff
@@ -0,0 +1,51 @@
+import { Button } from '@wordpress/components';
+import { check } from '@wordpress/icons';
+import { useTranslate } from 'i18n-calypso';
+import P2StepWrapper from 'calypso/signup/p2-step-wrapper';
+import './style.scss';
+
+function P2EmailConfirmed( {
+	flowName,
+	goToNextStep,
+	stepName,
+	submitSignupStep,
+	positionInFlow,
+} ) {
+	const translate = useTranslate();
+
+	const handleNextStepClick = ( option ) => {
+		submitSignupStep( {
+			stepName,
+			option,
+		} );
+
+		goToNextStep();
+	};
+
+	return (
+		<P2StepWrapper
+			flowName={ flowName }
+			stepName={ stepName }
+			positionInFlow={ positionInFlow }
+			headerIcon={ check }
+			headerText={ translate( 'Email confirmed' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **28** times:
`translate( 'Email confirmed!' )` *ES Score: 10*